config/global.ini.php

```diff
@@ -773,6 +773,9 @@
 ; By setting this option to 0, you can prevent Super User from editing the Geolocation settings.
 enable_geolocation_admin = 1
 
+; By setting this option to 0, you can prevent the geolocator to fall back to the default provider, if the configured provider doesn't provide any results.
```

Comment:
haven't reviewed the PR or tested anything. Just wanted to mention that reading this config name and description it wasn't clear what it does or means. Maybe we can partially rename the setting name and/or the description? What does it do to the data or to the user when disabling this setting?

Reply:
What about `enabled_geolocation_guess_country_fallback`?

Reply:
👍  better

Reply:
adjusted.